lib/Client.js

```diff
@@ -1387,16 +1387,33 @@ class Client extends EventEmitter {
 
     /**
     * Create a guild
-    * @arg {String} name The name of the guild
-    * @arg {String} region The region of the guild
-    * @arg {String} [icon] The guild icon as a base64 data URI. Note: base64 strings alone are not base64 data URI strings
+    * @arg {Object} options The properties of the guild
+    * @arg {String} options.name The name of the guild
```

Comment:
You documented name as a part of options object, while you used name as the first param